cg/apps/sequencing_metrics_parser/parsers/bcl2fastq.py

```diff
@@ -15,37 +18,46 @@
 LOG = logging.getLogger(__name__)
 
 
-def parse_bcl2fastq_sequencing_metrics(flow_cell_dir: Path) -> List[Bcl2FastqSampleLaneMetrics]:
+def parse_bcl2fastq_sequencing_metrics(flow_cell_dir: Path) -> List[SampleLaneMetrics]:
     """Parse metrics for a flow cell demultiplexed with Bcl2fastq."""
-    tile_sequencing_metrics: List[
-        Bcl2FastqSampleLaneTileMetrics
-    ] = parse_bcl2fastq_raw_tile_metrics(flow_cell_dir)
+    tile_metrics: List[SampleLaneTileMetrics] = parse_raw_tile_metrics(flow_cell_dir)
+    return combine_tile_metrics_per_sample_and_lane(tile_metrics)
 
-    return aggregate_tile_metrics_per_sample_and_lane(tile_sequencing_metrics)
 
-
-def parse_bcl2fastq_raw_tile_metrics(
+def parse_raw_tile_metrics(
     demultiplex_result_directory: Path,
-) -> List[Bcl2FastqSampleLaneTileMetrics]:
+) -> List[SampleLaneTileMetrics]:
     """Parse metrics for each tile on a flow cell demultiplexed with Bcl2fastq."""
-    tile_sequencing_metrics: List[Bcl2FastqSampleLaneTileMetrics] = []
+    tile_metrics: List[SampleLaneTileMetrics] = []
 
-    stats_json_paths: List[Path] = get_bcl2fastq_stats_paths(
-        demultiplex_result_directory=demultiplex_result_directory
-    )
+    stats_json_paths: List[Path] = get_bcl2fastq_stats_paths(demultiplex_result_directory)
 
     for stats_json_path in stats_json_paths:
         LOG.debug(f"Parsing stats.json file {stats_json_path}")
-        sequencing_metrics = Bcl2FastqSampleLaneTileMetrics.parse_file(stats_json_path)
-        tile_sequencing_metrics.append(sequencing_metrics)
+        metrics = SampleLaneTileMetrics.parse_file(stats_json_path)
+        tile_metrics.append(metrics)
+
+    return tile_metrics
+
 
-    return tile_sequencing_metrics
+def update_lane_metrics_with_demux_data(
+    sample_lane_metrics: SampleLaneMetrics, demux_result: DemuxResult
+):
+    reads: int = demux_result.number_reads
+    sample_lane_metrics.total_reads += reads * 2
```

Comment:
```suggestion
    sample_lane_metrics.total_reads += get_total_reads()
```
Add a function with a docstring explaining why the multiplication is needed for this. So that it is easily traceable for other in future.